docs/connect-device-using-adb-wirelessly.md

```diff
@@ -0,0 +1,146 @@
+# Connecting real device using adb wirelessly
+
+## Introduction
+
+Connecting Android Debug Bridge (ADB) wirelessly to your device can be convenient, especially when working with devices that don't have easily accessible USB ports or when you want to avoid cable clutter. This guide will walk you through the process of connecting ADB to your Android device wirelessly using the Command Line Interface (CLI).
+
+## Prerequisites
+
+- Android device
+- [Developer Options](https://developer.android.com/studio/debug/dev-options) enabled on your device.
+- Install Android SDK using [mobile-helper-tool](https://github.com/nightwatchjs/mobile-helper-tool): `npx @nightwatch/mobile-helper android`.
+
+### Android Debug Bridge (adb)
+
+The whole process is done using `adb` command line tool. So, you would need to make sure that `adb` is available from your terminal. You can check this by running `adb --version`.
+
+If `adb` is not available directly, you can either add its location to your `PATH` environment variable, or `cd` to the location where the `adb` binary is present and use it directly from there.
+
+The `adb` binary will be present in the `platform-tools` sub-directory of your Android SDK setup location.
+
+### Adding `adb` location to `PATH`
+
+Add the path of `platform-tools` directory to your `PATH` environment variable.
+
+**Linux/Mac**:
+
+Add the below command at the end of your `~/.bashrc` or `~./bash_profile` file and restart the terminal.
+
+```bash
+export PATH=$PATH:/path/to/Android/sdk/platform-tools/
+```
+
+**Windows**:
+
+Add the below path to your `PATH` environment variable in the Control Panel and restart the terminal.
+
+```bash
+\path\to\Android\sdk\platform-tools\
+```
+
+### Using `adb` directly
+
+To use `adb` directly (without adding it to `PATH`), simply go to the directly where the binary is present and use it as follows:
+
+```bash
+cd /path/to/Android/sdk/platform-tools/
+
+# for windows
+adb.exe --version
+
+# for mac/linux
+./adb --version
+```
+
+## Steps for Android v11 or above
+
+### Enable Wireless Debugging on your device
+
+1. Connect your device to the same network as your computer. You may connect the device to your computer's hotspot.
+2. Enable wireless debugging on your device by going to: `settings > wireless debugging`.
```

Comment:
We should also add a point for activating the Developer Options setting here by clicking on the Build Number, as done in the other PR. And also searching for "wireless debugging" in the Settings app as a secondary option to `settings > wireless debugging`.